packages/framework-types/src/concepts/filter-hooks.ts

```diff
@@ -0,0 +1,8 @@
+import { UserEnvelope } from '../envelope'
+import { CommandInput } from './command'
+
+export interface FilterHooks {
```

Comment:
As the function this interface represents is tightly coupled with Commands, we can call it `CommandFilterHooks`. Also, we can call the "FilterHooks" we are using in read models as `ReadModelFilterHooks`, as it is only useful for read models (because of the parameters the "before" function receives).

What do you think?